### DISCUSSION TITLE
Support for the experimental syntax 'jsx' isn't currently enabled

### DISCUSSION THREAD

**Discussion by maxcww** [TanStack/table#2767] [Q&A] [answered]
Hello,

I have a react project setup with create-react-app. I have also installed this react-table library, but the moment I try to use the argument useSortBy I get the following error:

> SyntaxError: /Users/max/Surmount/repos/ahv_django/ahv_react/node_modules/react-table/src/publicUtils.js: Support for the experimental syntax 'jsx' isn't currently enabled (10:36):
>
>  8 | 
>  9 | export const defaultRenderer = ({ value = '' }) => value;
>\>10 | export const emptyRenderer = () => <>&nbsp;</>;
>      |                                    ^
>  11 | 
>  12 | export const defaultColumn = {
>  13 |   Cell: defaultRenderer,
>
>Add @babel/preset-react (https://git.io/JfeDR) to the 'presets' section of your Babel config to enable transformation.
>If you want to leave it as-is, add @babel/plugin-syntax-jsx (https://git.io/vb4yA) to the 'plugins' section to enable parsing.

I do not know how to fix this error without ejecting my app from create-react-app which allows me to change the Babel config. Is there a way?

**Comment by Pavel1114** [comment, marked as answer]
I have solved same problem. The is in import statement. Pycharm's autoimport inserts
`import { useTable } from "react-table/src/hooks/useTable";`
but correct import is 
`import { useTable } from "react-table";

**Reply by dasler-fw** [reply]
I've been looking for a solution for a long time. Thank you very much @Pavel1114
